DEVELOPMENT.md

```diff
@@ -46,14 +86,16 @@ If you need to bump the major version, mark it as breaking change in the body of
 ```text
 fix: upgrade dependency X
 
-BREAKING CHANGE: requires minimum Node.js 14 to run
+BREAKING CHANGE: requires minimum Node.js 16 to run
 ```
 
-3. New versions of this action will be released automatically by the CI, see [.github/workflows/main.yml](.github/workflows/main.yml). This will create a new [GitHub release](https://github.com/cypress-io/github-action/releases) and will update the current `/v5` branch. Thus specifying `- uses: cypress-io/github-action@v5` selects the new version automatically.
+3. New versions of this action will be released automatically by the CI, see [.github/workflows/main.yml](.github/workflows/main.yml). This will create a new [GitHub release](https://github.com/cypress-io/github-action/releases) and will update the current `v5` branch. Thus specifying `uses: cypress-io/github-action@v5` selects the new version automatically.
```

Comment:
Is this on merge to master?

Reply:
@emilyrohrbough 
> Is this on merge to master?

Yes.

The `main` workflow trigers on pushes to the `master` branch or on `pull_request`. The `release` job runs only on the `master` branch in the parent repository (not in a fork). `cycjimmy/semantic-release-action@v3` is configured to run from `master`.